### PR TITLE
New recipe for MicroHapDB 0.2

### PR DESCRIPTION
**Pull request by standage**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* <strike>[ ] This PR updates an existing recipe.</strike>
* <strike>[ ] This PR does something else (explain below).</strike>

[MicroHapDB](https://github.com/bioforensics/MicroHapDB) version 0.2 was just released, and I'd like to add this to bioconda.

### CODE REVIEW

**Comment by standage** [comment]
The linter's error message isn't too clear here. :(

File: recipes/microhapdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+source:
+  url: https://github.com/bioforensics/microhapdb/archive/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: python
+  entry_points:
+    - microhapdb = microhapdb.cli:main
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 1
```
Comment (druvus) [inline recipes/microhapdb/meta.yaml]: You should start at 0 for new versions

File: recipes/microhapdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
```
Comment (druvus) [inline recipes/microhapdb/meta.yaml]: I think you may take away setuptools now when you are using pip.

Comment (standage) [reply recipes/microhapdb/meta.yaml]: Confirmed.

File: recipes/microhapdb/meta.yaml (hunk 1 above)
Comment (standage) [reply recipes/microhapdb/meta.yaml]: Thanks for the tip. I cobbled this together by looking at a few examples, and I'm still trying to work out the details. :-)

**Comment by standage** [comment]
🎉🎉🎉

**Comment by standage** [comment]
Thanks @druvus!

**Pull request merged**